geode-core/src/integrationTest/java/org/apache/geode/internal/cache/tier/sockets/CacheClientNotifierIntegrationTest.java

```diff
@@ -156,23 +171,21 @@ public void testCacheClientNotifier_NotifyClients_QRMCausesPrematureRemovalFromH
       return false;
     });
 
-    Assert.assertEquals("Expected the HAContainer to be empty", 0,
-        cacheClientNotifier.getHaContainer().size());
+    assertThat(cacheClientNotifier.getHaContainer()).as("Expected the HAContainer to be empty")
+        .isEmpty();
   }
 
   private HARegion createMockHARegion(InternalCache internalCache,
```

Comment:
while cleaning things here, it might be nice to make this void since we don't care or use `HARegion` result

Reply:
Changed createMockHARegion to void